common/src/main/java/apoc/result/IndexConstraintRelationshipInfo.java

```diff
@@ -9,16 +9,19 @@ public class IndexConstraintRelationshipInfo {
 
     public final String name;
 
-    public final Object type;
+    public final String type;
 
     public final List<String> properties;
 
     public final String status;
+    
+    public final Object relationshipType;
```

Comment:
Why is this an Object? (I see it was before as well), but is it not just a string?

Reply:
Because `relationshipType` can also be a `List<String>`, in case of composite indexes, e.g. in `testIndexesWithMultipleLabelsAndRelTypes` https://github.com/neo4j/apoc/pull/247/files/9b4b7bf4448cf27d6daff1f1bc9a9d44e23aa11e#diff-231d1cab850aaf45247b0fd9e1a64e18a8002685841df498e2098c38bedf6fb3R758

Reply:
Ah makes sense, thanks :)